.github/workflows/Operator-CI.yaml

```diff
@@ -0,0 +1,67 @@
+name: Shard Operator - CI
+on:
+  pull_request:
+    paths:
+      - 'shard-operator/**'
+jobs:
+  event-bridge-build:
+    env:
+      MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.showDateTime=true -Djava.awt.headless=true -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
+      NAMESPACE: mynamespace
+    concurrency:
+      group: event-bridge-operator-pr-${{ github.head_ref }}
+      cancel-in-progress: true
+    timeout-minutes: 45
+    runs-on: ubuntu-latest
+    name: Build and Test Operator
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache Maven Packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
```

Comment:
There are overlaps between this build and the other existing [CI.yaml file](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/CI.yaml#L31-L42).
Maven version for example here is not specified while in the other job it is so we can centralise this handling.
I think we can (with another ticket) split them in reusable composite actions similar like [this](https://github.com/kiegroup/kogito-pipelines/blob/main/.ci/actions/maven/action.yml) and the [corresponding usage](https://github.com/kiegroup/kogito-runtimes/blob/main/.github/workflows/kogito-runtimes-pr.yml#L36-L41).
Wdyt?
/cc @r00ta @robpblake

Reply:
Sounds good to me

Reply:
Reported as [MGDOBR-153](https://issues.redhat.com/browse/MGDOBR-153)